src/components/NetworkPanel/CyjsRenderer/CyjsRenderer.tsx

```diff
@@ -692,11 +692,17 @@ const CyjsRenderer = ({
     }
   }, [])
 
+  useEffect(() => {
+    if (cy !== null) {
+      renderNetwork(cy, networkView, displayMode, true)
+    }
+  }, [cy])
+
   useEffect(() => {
     if (cy !== null) {
       renderNetwork(cy, networkView, displayMode, false)
     }
-  }, [cy, networkView, displayMode])
+  }, [networkView, displayMode])
```

Comment:
@keiono This may need your further review. Not sure whether it is proper to do so.

Reply:
Applying fit function only at the initialization should be OK.